promql/fuzz.go

```diff
@@ -0,0 +1,67 @@
+package promql
+// +build gofuzz
+
+/* PromQL parser fuzzing instrumentation for use with https://github.com/dvyukov/go-fuzz.
+ * 
+ * Fuzz each parser by building appropriately instrumented parser, ex. FuzzParseMetric and execute it with it's 
+ *
+ *     go-fuzz-build -func FuzzParseMetric -o FuzzParseMetric.zip github.com/prometheus/prometheus/promql
+ *
+ * And then run the tests with the appropriate inputs
+ *
+ *     go-fuzz -bin FuzzParseMetric.zip -workdir fuzz-data/ParseMetric
+ *
+ * Further input samples should go in the folders fuzz-data/ParseMetric/corpus.
+ *
+ * Repeat for ParseMetricSeletion, ParseExpr and ParseStmt
+ */
+
+const (
+	fuzz_interesting = 1
+	fuzz_meh         = 0
+	fuzz_discard     = -1
+)
+
+// Fuzz the metric parser
```

Comment:
Please end sentence-style doc-comments with a period (https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences). Golint doesn't care about `Test...` not having proper doc comments, but probably the `Fuzz` ones would be complained about?